### ISSUE TITLE
Detect and report assert statements that always evaluate to true

### ISSUE THREAD

**Issue by erictraut** [microsoft/pyright#336]
Assert statements in python are sometimes errantly written such that they always evaluate to true. For example, this assert is legitimate:
```
assert really_long_variable_name  == False, "Error message"
```
But if you then decide to split it across lines by adding parens:
```
assert (really_long_variable_name  == False, 
    "Error message")
```
This is not what's intended because it will always evaluate to a non-empty tuple, and the assert will always fire.

**Comment by erictraut** [comment]
This will be included in the next release.

**Comment by erictraut** [comment]
This is now fixed in v1.1.1, which I just published.
